core/coins/format.go

```diff
@@ -10,53 +10,67 @@ import (
 	"cosmossdk.io/math"
 )
 
-// formatCoin formats a sdk.Coin into a value-rendered string, using the
-// given metadata about the denom. It returns the formatted coin string, the
-// display denom, and an optional error.
-func formatCoin(coin *basev1beta1.Coin, metadata *bankv1beta1.Metadata) (string, error) {
+const EmptyCoins = "empty coins"
```

Comment:
```suggestion
const EmptyCoins = "zero"
```

This was discussed in the Textual WG, and "coins" word might not be needed